lib/use-cases/RejectDropbox.js

```diff
@@ -1,10 +1,19 @@
-module.exports = ({ gateways: { dropbox: dropboxes } }) => {
+module.exports = ({ gateways: { dropbox: dropboxes, notify } }) => {
   return async ({ dropboxId, rejectReason }) => {
     try {
       const dropbox = await dropboxes.get(dropboxId);
       if (!dropbox) throw new Error('Invalid dropbox');
+
+      await notify.send({
+        email: dropbox.customer.parentsEmail,
+        referenceId: dropboxId,
+        templateId: process.env.GOV_NOTIFY_REJECTION_TEMPLATE_ID,
+        templateVars: { rejectReason }
+      });
+
```

Comment:
maybe save before sending? (Not a big deal tho)

Reply:
I thought I'd save after, so that if there was an error sending it would stay unrejected. Otherwise the parent would never be notified (ie. the staff member would then have to reject again)